sample/src/main/java/org/aerogear/graphqlandroid/adapter/TaskAdapter.kt

```diff
@@ -41,36 +39,35 @@ class TaskAdapter(private val notes: List<UserOutput>, private val context: Cont
     override fun onBindViewHolder(holder: TaskHolder, position: Int) {
 
         val currentTask = notes[position]
-        var title_task = currentTask.title
-        var desc_task = currentTask.desc
-        var id_task = currentTask.id.toString()
+        val titleTask = currentTask.title
+        val descTask = currentTask.desc
+        val idTask = currentTask.id.toString()
 
         with(holder.itemView) {
             user_switch.isChecked = false
             title_tv.text = currentTask.title
             desc_tv.text = currentTask.desc
             id_tv.text = currentTask.id.toString()
-            if (!currentTask.firstName.isNullOrBlank()) {
+            if (!currentTask.firstName.isBlank()) {
```

Comment:
We need to check for the nullanblity of the firstName as well, might want to add that back in :)